airbyte-cdk/bulk/toolkits/load-avro/src/testFixtures/kotlin/io/airbyte/cdk/load/data/avro/AvroRecordToAirbyteValue.kt

```diff
@@ -74,10 +72,10 @@ class AvroRecordToAirbyteValue {
                 )
             is IntegerType -> return IntegerValue(avroValue as Long)
             is NumberType -> return NumberValue((avroValue as Double).toBigDecimal())
-            is ObjectTypeWithEmptySchema ->
-                throw UnsupportedOperationException("ObjectTypeWithEmptySchema is not supported")
-            is ObjectTypeWithoutSchema ->
-                throw UnsupportedOperationException("ObjectTypeWithoutSchema is not supported")
+            is UnknownType,
+            ArrayTypeWithoutSchema,
+            ObjectTypeWithEmptySchema,
+            ObjectTypeWithoutSchema,
             is StringType ->
```

Comment:
Does this mean UnknownType, ArrayTypeWithoutSchema, ObjectTypeWithEmptySchema, ObjectTypeWithoutSchema fall into the below string type block?

Reply:
yes; they are all converted to strings